server/core/hot_region_storage_test.go

```diff
@@ -63,8 +63,8 @@ func (m *MockPackHotRegionInfo) GenHistoryHotRegions(num int, updateTime time.Ti
 			FlowBytes:     rand.Float64() * 100,
 			KeyRate:       rand.Float64() * 100,
 			QueryRate:     rand.Float64() * 100,
-			StartKey:      []byte(fmt.Sprintf("%20d", i)),
```

Comment:
Do we need more tests to test key format?

Reply:
I add some test about key format

Reply:
this function is used by benchmark,not test